network.tf

```diff
@@ -0,0 +1,109 @@
+
+
+resource "vcd_nsxt_app_port_profile" "k8sapi" {
+
+  name        = "k8sapi"
+  description = "Application port profile kubenetes api, with TKG"
+  scope = "TENANT"
+
+  app_port {
+    protocol = "TCP"
+    port     = ["6443"]
+  }
+  lifecycle {
```

Comment:
lifecycle:-
May not be required, was testing issue where profile is not cleaned up overnight. 10.4 will fix this I believe.